bundles/org.openhab.io.homekit/README.md

```diff
@@ -836,6 +839,33 @@ Number          cooler_cool_thrs           "Cooler Cool Threshold Temp [%.1f °C
 Number          cooler_heat_thrs           "Cooler Heat Threshold Temp [%.1f °C]" (gCooler)            {homekit="HeatingThresholdTemperature" [minValue=0.5, maxValue=20]}
 ```
 
+## Multiple Instances
+
+Homekit has a limitation of 150 accessories per bridge. The bridge itself
+counts as an accessory, so in practice it's 149. In order to overcome this
+limitation, you can instruct OpenHAB to expose multiple bridges to
+Homekit, and then manually assign specific accessories to different
+instances. You will need to manually add each additional bridge in the
+Home app, since the QR Code in settings will only be for the primary bridge;
+however the same PIN is still used. In order to assign a particular
+accessory to a different bridge, set the `instance` metadata parameter:
```

Comment:
Our convention for maintainability is to return to a new line at the end of a sentence, not inside a sentence.
Please also fix your other addings in README.